mono/metadata/reflection-internals.h

```diff
@@ -41,7 +41,7 @@ MonoType *
 mono_reflection_type_handle_mono_type (MonoReflectionTypeHandle ref_type, MonoError *error);
 
 MonoType*
-mono_reflection_get_type_checked (MonoImage *rootimage, MonoImage* image, MonoTypeNameParse *info, mono_bool ignorecase, mono_bool *type_resolve, MonoError *error);
+mono_reflection_get_type_checked (MonoImage *rootimage, MonoImage* image, MonoTypeNameParse *info, mono_bool ignorecase, mono_bool *type_resolve, gboolean searchMscorlib, MonoError *error);
```

Comment:
I need to go through the whole thing in mode detail, but two quick nits: s/`searchMscorlib`/`search_mscorlib` - mono uses lowercase and underscores, mostly.  Aand: `type_resolve` is an out argument (I think...) so I think `search_mscorlib` should come before it.

Reply:
Yeah whoops, not sure what I was thinking on the name. Updated.